[32mINFO    [0m integration.ha.test_backups:test_backups.py:490 Remove s3-credentials relation
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:502 Re-add s3-credentials relation
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [executing] maintenance: Configuring backup service...
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:511 Syncing credentials for aws
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
  s3-integrator/0 [idle] active: 
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.17.149.33:9200/_recovery?human
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=338, count=339, last_expected_id=338)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.17.149.33:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.17.149.33:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.17.149.33:9200/series_index/_count
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.17.149.33:9200/_recovery?human
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.17.149.33:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.17.149.33:9200/series_index/_count
[32mINFO    [0m integration.ha.helpers:helpers.py:591 Testing restore for series_index - original count pre-restore: 339, and now, new count: 308
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.17.149.33:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.17.149.33:9200/series_index/_count
[32mINFO    [0m integration.ha.conftest:conftest.py:39 



The writes have been cleared.




[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  02:09:19Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active      2  opensearch                                 0  no       
main                               active      1  opensearch                                 2  no       
opensearch                         active      1  opensearch                                 1  no       
s3-integrator                      active      1  s3-integrator             latest/edge     49  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
failover/0                   active    idle   3        10.17.149.84    9200/tcp  
failover/1*                  active    idle   4        10.17.149.118   9200/tcp  
main/0*                      active    idle   5        10.17.149.33    9200/tcp  
opensearch/0*                active    idle   2        10.17.149.86    9200/tcp  
s3-integrator/0*             active    idle   0        10.17.149.52              
self-signed-certificates/0*  active    idle   1        10.17.149.74              

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.17.149.52   juju-80e0ef-0  ubuntu@22.04      Running
1        started  10.17.149.74   juju-80e0ef-1  ubuntu@22.04      Running
2        started  10.17.149.86   juju-80e0ef-2  ubuntu@22.04      Running
3        started  10.17.149.84   juju-80e0ef-3  ubuntu@22.04      Running
4        started  10.17.149.118  juju-80e0ef-4  ubuntu@22.04      Running
5        started  10.17.149.33   juju-80e0ef-5  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-1: 01:56:24 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 01:56:24 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:56:24 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:56:24 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-s3-integrator-0: 01:56:25 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-self-signed-certificates-0: 01:56:25 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 01:56:46 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 01:56:46 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-1: 01:56:46 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 01:56:49 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 01:56:49 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 01:56:49 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-5: 01:56:51 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 01:56:51 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:56:51 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:56:51 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-main-0: 01:56:51 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-failover-0: 01:56:51 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-main-0: 01:58:40 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:58:41 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:58:41 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:58:44 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:58:47 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:58:50 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-main-0: 01:58:53 ERROR unit.main/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-failover-0: 01:59:35 ERROR unit.failover/0.juju-log certificates:14: Cannot connect to the OpenSearch server...
unit-failover-0: 01:59:36 ERROR unit.failover/0.juju-log certificates:14: Cannot connect to the OpenSearch server...
unit-failover-1: 01:59:37 ERROR unit.failover/1.juju-log opensearch-peers:6: Cannot connect to the OpenSearch server...
unit-failover-1: 01:59:38 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:59:40 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:59:41 ERROR unit.failover/1.juju-log opensearch-peers:6: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:59:43 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:59:44 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:59:45 ERROR unit.opensearch/0.juju-log peer-cluster:12: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:59:46 ERROR unit.opensearch/0.juju-log opensearch-peers:2: Cannot connect to the OpenSearch server...
unit-failover-1: 01:59:56 ERROR unit.failover/1.juju-log upgrade-version-a:5: Cannot connect to the OpenSearch server...
unit-failover-1: 01:59:58 ERROR unit.failover/1.juju-log upgrade-version-a:5: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:00:19 ERROR unit.opensearch/0.juju-log peer-cluster:12: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:00:20 ERROR unit.opensearch/0.juju-log peer-cluster:12: Cannot connect to the OpenSearch server...
unit-main-0: 02:02:26 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 02:02:44 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-failover-1: 02:03:07 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:03:07 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:03:09 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:03:09 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:03:10 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:03:11 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:03:11 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:03:12 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:03:13 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:03:13 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:03:15 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:03:15 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...
[32mINFO    [0m integration.ha.test_backups:test_backups.py:173 Cleaning backups from cloud buckets